src/handler.js

```diff
@@ -55,44 +55,66 @@ class Handler {
       if (decider !== 'any' && status.status.state !== decider) {
         return;
       }
+
+      let ircmessage = `Task "${task.metadata.name}" complete with status '${status.status.state}'. Inspect: ${href}`;
+
       switch (route[1]) {
         case 'irc-user':
           this.monitor.count('notification-requested.irc-user');
+          if (_.has(task, 'extra.notify.irc-user.message')) {
```

Comment:
`irc.userMessage` or `ircUserMessage` or `ircUser.Message`

This should not be the same as the routes...